src/test/kotlin/org/rust/ide/navigation/goto/RustGotoSuperHandlerTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.rust.ide.navigation.goto
+
+import com.intellij.lang.CodeInsightActions
+import org.rust.lang.RustLanguage
+import org.rust.lang.RustTestCaseBase
+
+class RustGotoSuperHandlerTest : RustTestCaseBase() {
+    override val dataPath = "org/rust/ide/navigation/fixtures/goto_super"
+
+    fun testGotoSuper() {
+        myFixture.copyFileToProject("main.rs", "/main.rs")
```

Comment:
You should also remove the `main.rs` file itself :)

You don't need to setup targets here at all, because your whole test is inside a single file, and we are able to do resolve within a sing file without the knowledge of a cargo project